portfolio/src/main/webapp/script.js

```diff
@@ -12,31 +12,91 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+window.addEventListener('DOMContentLoaded', () => {
+  const navBar = document.getElementById('nav-bar');
+
+  Array.from(navBar.children).forEach(child => {
+      child.addEventListener('click', () => {
+          const targetId = child.getAttribute('data-target-id');
+          const targetElement = document.getElementById(targetId);
+          showContent(targetElement);
+        });
+      });
+
+  document.getElementById("comment-amount")
+      .addEventListener("change", getComments);
+    
+  document.getElementById("delete-comments")
+      .addEventListener("click", deleteComments);
+
+  getComments();
+});
+
+/** Displays the content title and text in the content container. */
+function showContent(element) {
+  const contentContainer = document.getElementById('content-container');
+
+  Array.from(contentContainer.children).forEach(child => {
+      child.classList.toggle('hidden', true);
+  });
+
+  element.classList.toggle('hidden', false);
+}
+
 /** Retrieves data from the /data page and displays it. */
 function getComments() {
-  fetch('/data').then(response => response.json()).then((json) => {
-    json.forEach(comment => displayComment(comment));
-  });
+  clearComments();
+
+  localStorage.setItem("commentAmount",
```

Comment:
This is always just going to use whatever value is in "comment-amount" and not really what is in `localStorage`.

The "comment-amount" input needs to be populated with the value from `localStorage` in 'DOMContentLoaded'.

Reply:
Done.